tee-worker/litentry/core/stf-task/sender/src/lib.rs

```diff
@@ -67,13 +68,15 @@ use sp_std::prelude::Vec;
 /// https://www.notion.so/web3builders/Sidechain-block-importer-and-block-production-28292233b4c74f4ab8110a0014f8d9df
 
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
-pub struct IdentityVerificationRequest {
+pub struct IdentityLinkRequest {
```

Comment:
From my understanding we are still requesting verification, and if its ok then linking is performed.

Reply:
Reverted!